wallets/wrap.js

```diff
@@ -10,21 +9,28 @@ const INCOMING_EXPIRATION_BUFFER_MSECS = 300_000 // the buffer enforce for the i
 const MAX_OUTGOING_CLTV_DELTA = 500 // the maximum cltv delta we'll allow for the outgoing invoice
 export const MIN_SETTLEMENT_CLTV_DELTA = 80 // the minimum blocks we'll leave for settling the incoming invoice
 const FEE_ESTIMATE_TIMEOUT_SECS = 5 // the timeout for the fee estimate request
+const MAX_FEE_ESTIMATE_PERCENT = 3n // the maximum fee relative to outgoing we'll allow for the fee estimate
```

Comment:
isn't it better to progressively move all the constants under lib/constants so we can see and tweak all of them in a single file?

Reply:
imho if the constants will only be used in one place, they should be defined in the same place they are used. It's a code locality thing.

The idiom I'm familiar with is "define a variable closest to where it is used"